src/visitor/CollectAllExprsVisitor.h

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef VISITOR_COLLECTALLEXPRSVISITOR_H_
+#define VISITOR_COLLECTALLEXPRSVISITOR_H_
+
+#include <unordered_set>
+
+#include "common/expression/Expression.h"
+#include "visitor/ExprVisitorImpl.h"
+
+namespace nebula {
+namespace graph {
+
+class CollectAllExprsVisitor final : public ExprVisitorImpl {
+public:
+    explicit CollectAllExprsVisitor(const std::unordered_set<Expression::Kind>& exprKinds);
+    bool ok() const override {
+        return true;
```

Comment:
return !exprs_.empty();

Reply:
here should always return TRUE for traversing all nodes in expression tree.